runtime/etc/Makefile.comm-ugni

```diff
@@ -0,0 +1,37 @@
+# Copyright 2004-2017 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Conservatively use CXX as the linker, in case regexp (or other C++
+# code) is being linked in.
+#
+LD = $(CXX)
+
+#
+# If there isn't a hugepage module loaded, we need to request
+# libhugetlbfs ourselves.
+#
+ifeq (,$(findstring HUGETLB,$(PE_PRODUCT_LIST)))
+GEN_LFLAGS += -lhugetlbfs
+endif
+
+#
+# CCE doesn't understand -pthread, and the cc wrapper already throws -lpthread
```

Comment:
Maybe remove 'the cc wrapper', just to prevent anyone from over-generalizing the second part of the sentence to apply to all cc command line drivers, not just the CCE one.